code/modules/cargo/packs/engineering.dm

```diff
@@ -73,7 +73,7 @@
 	desc = "No rechargers? No problem, with the NT-75 EPI, you can recharge any standard \
 		cell-based equipment anytime, anywhere. Contains two Inducers."
 	cost = CARGO_CRATE_VALUE * 4
-	contains = list(/obj/item/inducer/sci {cell_type = /obj/item/stock_parts/cell/inducer_supply; opened = 0} = 2) //FALSE doesn't work in modified type paths apparently.
+	contains = list(/obj/item/inducer {cell_type = /obj/item/stock_parts/cell/inducer_supply; opened = 0} = 2) //FALSE doesn't work in modified type paths apparently.
```

Comment:
man i fucking hate how we're singletonning the type in here, can you just make a subtype of `/obj/item/inducer` so we don't have to deal with this bullshit. call it `/obj/item/inducer/cargo`

Reply:
Ok, should be less ugly now.